### ISSUE TITLE
CONFIGURE: Python system pacakge install to check if already installed

### ISSUE THREAD

**Issue by gjwgit** [mlhubber/mlhub#82]
Perhaps could reuse system.sh to install the python system dependencies?
```
$ ml configure sar
[...]
*** Installing system Python package python3-sklearn ...
Reading package lists... Done
Building dependency tree       
Reading state information... Done
python3-sklearn is already the newest version (0.19.1-3).
0 to upgrade, 0 to newly install, 0 to remove and 65 not to upgrade.
[...]

$ dpkg-query -s python3-sklearn
Package: python3-sklearn
Status: install ok installed
[...]
```

**Comment by simonzhaoms** [comment]
Good idea😁

**Comment by simonzhaoms** [comment]
Solved in commit 24acac8
